src/WickrIOConfigure.js

```diff
@@ -537,7 +573,9 @@ class WickrIOConfigure
     {
         try {
             await this.inputTokens(integrationName);
-            console.log("Finished Configuring!");
+            if(this.addOnToJSON === true){
```

Comment:
So if addOnToJson is false will never see this log statement.  There is probably only one integration that will currently use the addOnToJson.

I'd just get rid of the if statement.  Always print out the "Finishing Configuration!"